func/HeroicBashLauncher.py

```diff
@@ -25,9 +23,9 @@
 
         #Don't create AddToSteam script if Steam Deck 
         if "deck" in os.path.expanduser("~"):
-            os.system('zenity --info --title="Process Finished" --text="Launch scripts stored in GameFiles folder\n\nYour games have been synced to Steam\n\nHave fun gaming!" --width=300')
+            zenity_popup(title="Process Finished", text="Launch scripts stored in GameFiles folder\n\nYour games have been synced to Steam\n\nHave fun gaming!")
```

Comment:
Will the zenity dialog be generated? Because line 8 in zenity.py says - use zenity dialog if `"deck" not in os.path.expanduser("~")`

Reply:
Ah ok, you're right. I think I misunderstood what these zenity popups are for so I need to rejig this MR a bit. Let me propose this:
1. If the user runs from the console directly then all zenity dialogs are disabled, the log output is printed in the console so they don't serve much benefit.
2. If the user runs from setup.sh (either in console, desktop link, etc.) then the zenity popups are the primary feedback since the console log is redirected to the log file.

Reply:
I'm okay with both of these.

Reply:
OK, after some messing about I realized that the paths are really wonky with the base/AppImage/Flatpak stuff, so I made it easier to generate that pyinstaller thing by introducing a makefile. You can still run that exe manually, with arguments and such, but the main use-case is definitely using that setup.sh script.